web/themes/custom/itkdev/itkdev_project_theme/templates/reports-project-track.html.twig

```diff
@@ -7,8 +7,17 @@
 {% extends '@itkdev_project_theme/content-wrapper.html.twig' %}
 
 {% block title %}
+  {% if data.projectTracks|length is same as(1) %}
```

Comment:
I know that using `same as` is the correct way to do a `===` comparison in Twig, but I find it much easier to read (and write) `1 == data.projectTracks|length`.